docs/version0.38.md

```diff
@@ -46,6 +47,12 @@ This option controls the handling of the `SIGUSR2` signal by the VM signal handl
 
 For more information, see [`-XX:[+|-]HandleSIGUSR2`](xxhandlesigusr2.md).
 
+### Technical preview of CRIU feature
+
+A technical preview of CRIU feature is included in this release. It is currently available for OpenJDK 11 and later running on Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only.
+
+Checkpoint Restore in User Space (CRIU) feature can be used to stop the VM at a particular checkpoint, save the state, and then enable the VM to resume from the point where it was stopped. You can restore the checkpoint image in as many environments as possible with improved start-up time and hence with better VM performance. For more information, see [CRIU feature](criufeature.md).
```

Comment:
"as many environments as possible" doesn't make much sense. Maybe replace with "as many other environments as you like" or "multiple other environments". I guess the aim here is to tell users that you can move the checkpoint image to somewhere else then run it, repeatedly. We should mention containers, since that is what CRIU was created for and what the OpenJ9 support is aimed at. Maybe something like "This capability means that the CRIU support is particularly useful in container environments."

Reply:
Done